tools/tojson.sh

```diff
@@ -1,5 +1,5 @@
 #/bin/bash
-# Copyright 2023 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
OSPO policy is not to update the year for copyright and just maintain it as the year it was created.

Reply:
Done